app/Http/Controllers/Task/TaskController.php

```diff
@@ -51,44 +55,53 @@ public function index()
         ], compact('list_tasks'));
     }
 
-    public function getEditTasks($id)
+    public function edit($id)
     {
-        try {
-            $task = $this->tasks->findOrFail($id);
-            $list_users = $this->users->getAllUser();
+        $task = $this->tasks->find($id);
+        $list_users = $this->users->getAllUser();
+        if ($task) {
             return view('tasks.edit_task', [
                 'idTasks' => $task,
                 'list_users' => $list_users
             ], compact('idTasks'));
-        } catch (ModelNotFoundException $e) {
+        } else
             return redirect()->back();
-        }
     }
 
-    public function postEditTasks(TaskRequest $request, $id)
+    public function update(TaskRequest $request, $id)
     {
-        try {
-            $task = $this->tasks->findOrFail($id);
-            $task->user_id = $request->userId;
-            $task->name = $request->name;
-            $task->description = $request->description;
-            $task->updated_at = Carbon::now();
-            if (!$task->save()) {
-                return redirect()->back()->with([
-                    trans('messages.edit_form.messageFail'),
-                    trans('messages.edit_form.classError'),
-                ]);
-            }
+        $data = $request->all();
+        $data->user_id = $request->userId;
+        $data->name = $request->name;
+        $data->description = $request->description;
+        $data->updated_at = Carbon::now();
+        $task = $this->tasks->update($id, $data);
```

Comment:
Từ dòng 74 đến 78. Đoạn này create task với tất cả attributes truyền lên thì làm luôn như này cũng đc em. Ngắn gọn
`$task = $this->tasks->update($id, $request->all());
Vì bên model có set $fillable rồi nên không sợ các attributes "laj" lọt vào e.
`Em thử xem có được k nhé. Check kỹ giúp a có chạy validate Ok không.